### PR TITLE
[W5.3b][T10-A3]Wang Jie

### PR DESCRIPTION
**Pull request by WangJieee**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+    public final String value;
+    private boolean isPrivate;
```
Comment (nguyenvanhoang7398) [inline src/seedu/addressbook/data/person/Contact.java]: maybe you can consider changing "isPrivate" attribute from "private" to "protected" cause there might be some cases when we need to access this attribute from Contact's children 😄

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    public Contact(String contact, boolean isPrivate){
+        String trimmedContact = contact.trim();
+        this.isPrivate = isPrivate;
+        this.value = trimmedContact;
+    }
+
+    public String toString() {
```
Comment (madsonic) [inline src/seedu/addressbook/data/person/Contact.java]: always annotate with `@Override` when overriding

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        return value;
+    }
+
+    public boolean equals(Object other) {
```
Comment (madsonic) [inline src/seedu/addressbook/data/person/Contact.java]: always annotate with `@Override` when overriding

File: src/seedu/addressbook/data/person/Contact.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        return other == this // short circuit if same object
+                || (other instanceof Contact // instanceof handles nulls
+                && this.value.equals(((Contact) other).value)); // state check
+    }
+
+    public int hashCode() {
```
Comment (madsonic) [inline src/seedu/addressbook/data/person/Contact.java]: always annotate with `@Override` when overriding

**Comment by madsonic** [comment]
Good job! with regards to `@Override` always use it so that the IDE checks that you are actually overriding a method. Otherwise, a typo in the method name for example could lead to hours of debugging when there is nothing wrong with the definition of the method. Imagine you type `equal` instead of `equals`!

**Pull request closed without merging**